test/integration/proxy_test.go

```diff
@@ -52,11 +53,14 @@ var _ = Describe("", Label("openshift-preset", "goproxy"), func() {
 			Expect(err).NotTo(HaveOccurred())
 
 			cmd := exec.Command("oc", "config", "use-context", "crc-admin")
-			err = cmd.Run()
+			out, err := cmd.Output()
+			if err != nil {
+				fmt.Printf("error running 'oc config use-context crc-admin' - output: %s\n", string(out))
```

Comment:
should we put that also as `logging.Debugf` instead `Printf` ?

Reply:
This is not meant to land in `main`, I'm trying to understand why the integration tests fail more often than not, the commit should be removed before pushing.

Reply:
I've removed these debug commits now